### PR TITLE
XS-3165: Update the DQC-0015's CSV's to V2

### PR DESCRIPTION
**Pull request by chrislococo-wf**
#### Changes:

- Updated the CSV files for 0015, due to them being out of sync with the documentation.

Please review: @hefischer  @andrewperkins-wf

### CODE REVIEW

**Comment by derekgengenbacher-wf** [comment]
+1

File: dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv [hunk 1]
```diff
@@ -6,34 +6,29 @@ BLE,Member,,Contains_insensitive,Netting
...
+BLE,Axis,,Equals,us-gaap:QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxi
 BLE,Axis,,Equals,us-gaap:PartnerCapitalComponentsAxis
 BLE,Axis,,Equals,us-gaap:ChangeInAccountingEstimateByTypeAxis
 BLE,Axis,,Equals,us-gaap:PartnerTypeOfPartnersCapitalAccountAxis
-BLE,Axis,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateAndOtherMember
-BLE,Axis,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateMember
-BLE,Axis,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:AllOtherSegmentsMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateAndOtherMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:CorporateMember
+BLE,Member,,Has_member,us-gaap:StatementBusinessSegmentsAxis|us-gaap:AllOtherSegmentsMember
```
Comment (mindeerickson-wf) [inline dqc_us_rules/resources/DQC_US_0015/dqc_15_exclusion_rules.csv]: Can you update the Jira ticket with the additions/deletions made to the exclusion file, similar to what you did for the concepts file?

**Comment by mindeerickson-wf** [comment]
Pass complete.

**Comment by davidtauriello** [comment]
This file - dqc_15_exclusion_rules.csv - needs an 's' at the end of line 28: QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxi

should be: QuantifyingMisstatementInCurrentYearFinancialStatementsByNatureOfErrorAxis

https://github.com/DataQualityCommittee/dqc_us_rules/blob/master/docs/DQC_US_0015/DQC_0015_MemberExclusions_V2.xlsx (row 28)

**Comment by derekgengenbacher-wf** [comment]
+1 on fixed typo

**Comment by mindeerickson-wf** [comment]
+1

**Comment by jasonleinbach-wf** [comment]
+1

**Comment by mindeerickson-wf** [comment]
Testing notes can be found in the Jira ticket.

QA +1

**Comment by andrewperkins-wf** [comment]
Merging - code changes are complete and testing is complete

**Pull request merged**